images/openshift-enterprise-hyperkube.yml

```diff
@@ -1,11 +1,14 @@
 container_yaml:
   go:
     modules:
-    - module: github.com/openshift/origin
+    - module: k8s.io/kubernetes
 content:
   source:
-    alias: ose
-    dockerfile: images/hyperkube/Dockerfile.rhel
+    dockerfile: openshift-hack/images/hyperkube/Dockerfile.rhel
+    git:
+      branch:
+        target: release-{MAJOR}.{MINOR}
+      url: git@github.com:openshift-priv/kubernetes.git
```

Comment:
you're going to want a `release-4.6` branch on this (I would expect it should get forwarded if you create it in https://github.com/openshift/kubernetes). once it has the dockerfile and specfile as specified we can run test builds.

Reply:
I've just created a release-4.6 branch in the openshift/kubernetes repo with the state I expect post-transition. Do you know how long it usually takes for changes to propagate to the openshift-priv equivalent?

Reply:
hopefully it's < 15min